.codecov.yml

```diff
@@ -0,0 +1,13 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+  status:
+    project:
+      default:
+        target: 70%    # the required coverage value
+        threshold: 1%  # the leniency in hitting the target
```

Comment:
1% is a big threshold to allow coverage to drop.  Would prefer a fraction of a percent like 0.2%.